functions/src/db/isTokenValidInDb.ts

```diff
@@ -0,0 +1,23 @@
+import { JWTData } from '../types/JWTData'
+import { JWTToken } from '../types/JWTToken'
+import { db } from '../firebase/firebase'
+import { UserData } from '../types/UserData'
+
+export const isTokenValidInDb = async (
+    jwtData: JWTData,
+    token: JWTToken
+): Promise<boolean> => {
+    const result = await db.collection('users').doc(jwtData.uid).get()
+
+    if (!result || !result.exists) {
```

Comment:
`if (!result?.exists)` if we're in node 14

Reply:
TS 4.0 does it too. thanks.